### ISSUE TITLE
Only working for Melle weapons

### ISSUE THREAD

**Issue by Scaper250** [BraveH/gear-switch-alert#16]
Trying to add opacity because I want to completely fill my weapons with their respective combat colour, but it's only working for melle, It goes fully red but the other cb styles don't

**Comment by tylercasper** [comment]
Yeah this just randomly doesn't work for some equipment. Just now it stopped working for my magic set.

**Comment by DeBelserArne** [comment]
Could it also have issues with Tumeken/Sang? Equiping these items when set to mage, act as melee? Is this related?
